client/src/components/profileInfoEdit/ProfileInfoEdit.js

```diff
@@ -21,22 +21,49 @@ const ProfileInfoEdit = () => {
   const dispatch = useDispatch();
 
   const {
-    teamId, teamName, orgName, email,
+    teamId, teamName, orgName, email, profilePic,
   } = useSelector(
     (state) => state.team,
   );
 
-  const [profileData, setInputs] = useState({ teamName, orgName, email });
+  const [profileData, setInputs] = useState({
```

Comment:
nit: rename the setter to `setProfileData` to be consistent with React `useState` naming conventions.